src/components/Movie/Movie.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import './Movie.css';
 
-const Movie = ({ poster }) => {
+const Movie = ({ poster, id }) => {
```

Comment:
Looks like the `id` isn't being used (yet). Did you mean to add it as an attribute on the `div` or the `img`?

This is reminding me that we still need to add the `alt` attribute on our images. I'll make an issue for it.

Reply:
I kind of through it in as a reminder that it will be utilized for iteration 2 when users click on a movie